src/hooks/useConceptName.ts

```diff
@@ -1,5 +1,6 @@
 import useSWR from 'swr';
 import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import { isEmpty } from '@openmrs/openmrs-form-engine-lib';
```

Comment:
Did you mean to import from lodash or some other lib? I think a null check would be sufficient here: `!conceptId`

Reply:
Done!